code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -80,7 +80,7 @@
 /datum/reagent/dylovene
 	name = "Dylovene"
 	description = "Dylovene is a broad-spectrum antitoxin used to neutralize poisons before they can do significant harm."
-	taste_description = "a roll of gauze"
+	taste_description = "a groll of gauze"
```

Comment:
???

the description was silly before, but i'm not sure this is much better

Reply:
I'm not sure how this made it in, actually. I'm 100% certain this is a typo I made during Reagent changes.